sql/moz-fx-data-shared-prod/firefox_desktop/dataset_metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Firefox Desktop
+description: |-
+  Data related to messaging_system.
```

Comment:
Is `messaging_system` correct here? What does cfr stand for?

Reply:
messaging_system is one of the system that used PingCentre and is being replaced by Glean.  There are different types of messages sent such as Onboarding (which includes Onboarding, messaging and communication)  and CFR (Contextual Feature Recommendation aka doorhanger). See  https://experimenter.info/messaging/desktop-messaging-surfaces#doorhanger